modules/qna/src/backend/index.ts

```diff
@@ -54,13 +54,33 @@ const onFlowChanged = async (bp: typeof sdk, botId: string, newFlow: sdk.Flow) =
   }
 }
 
+const onElementChanged = async (bp, botId, action, element) => {
+  const qnaStorage = await botScopedStorage.get(botId)
+  const questions = await qnaStorage.getQuestions({ question: '', categories: [] }, { limit: 0, offset: 0 })
```

Comment:
we could spare this call on all actions that are not `delete` if we moved this line inside the `if` ?

Reply:
Yes